onebusaway-gtfs-transformer/src/main/java/org/onebusaway/gtfs_transformer/factory/TransformFactory.java

```diff
@@ -806,4 +814,20 @@ private void configureStopNameUpdates(GtfsTransformer transformer, String path)
       throw new IllegalStateException(ex);
     }
   }
+
+  private String unescapeReplaceExpression(String value) {
```

Comment:
Can you make this static, package private and add a parameterised test with all all the expected inputs and outputs?